app/views/omni_nav_v2/_utility_nav.html.erb

```diff
@@ -39,23 +39,23 @@
                   </a></li>
-                  <li><a href="http://www.linkedin.com/company/10701?trk=tyah" target="_blank">
-                    <svg class="icon icon-linkedin2" xmlns="http://www.w3.org/2000/svg" width="32" height="32" viewBox="0 0 32 32"><title>  linkedin2</title><path d="M12 12h5.5v2.8h0.1c0.8-1.4 2.7-2.8 5.5-2.8 5.8 0 6.9 3.6 6.9 8.4v9.6h-5.8v-8.5c0-2 0-4.7-3-4.7-3 0-3.5 2.2-3.5 4.5v8.7h-5.8v-18z"/><path d="M2 12h6v18h-6v-18z"/><path d="M8 7c0 1.7-1.3 3-3 3s-3-1.3-3-3c0-1.7 1.3-3 3-3s3 1.3 3 3z"/></svg>
-                  </a></li>
                   <li><a href="http://itunes.apple.com/us/institution/chapman-university/id430678922" target="_blank">
                     <svg class="icon icon-graduation" xmlns="http://www.w3.org/2000/svg" width="32" height="32" viewBox="0 0 32 32"><title>youtube</title><path d="M31.7 9.6c0 0-0.3-2.2-1.3-3.2-1.2-1.3-2.6-1.3-3.2-1.4-4.5-0.3-11.2-0.3-11.2-0.3h0c0 0-6.7 0-11.2 0.3-0.6 0.1-2 0.1-3.2 1.4-1 1-1.3 3.2-1.3 3.2s-0.3 2.6-0.3 5.2v2.4c0 2.6 0.3 5.2 0.3 5.2s0.3 2.2 1.3 3.2c1.2 1.3 2.8 1.2 3.5 1.4 2.6 0.2 10.9 0.3 10.9 0.3s6.7 0 11.2-0.3c0.6-0.1 2-0.1 3.2-1.4 1-1 1.3-3.2 1.3-3.2s0.3-2.6 0.3-5.2v-2.4c0-2.6-0.3-5.2-0.3-5.2zM12.7 20.2v-9l8.6 4.5-8.6 4.5z"/></svg>
                   </a></li>
-                  <li><a href="http://pinterest.com/chapmanu/" target="_blank">
-                    <svg class="icon icon-pinterest" xmlns="http://www.w3.org/2000/svg" width="32" height="32" viewBox="0 0 32 32"><title>  pinterest2</title><path d="M16 0c-8.8 0-16 7.2-16 16s7.2 16 16 16 16-7.2 16-16-7.2-16-16-16zM16 29.9c-1.4 0-2.8-0.2-4.1-0.6 0.6-0.9 1.4-2.4 1.7-3.6 0.2-0.6 0.9-3.3 0.9-3.3 0.5 0.9 1.8 1.6 3.2 1.6 4.2 0 7.2-3.8 7.2-8.6 0-4.6-3.7-8-8.5-8-6 0-9.2 4-9.2 8.4 0 2 1.1 4.6 2.8 5.4 0.3 0.1 0.4 0.1 0.5-0.2 0-0.2 0.3-1.1 0.4-1.6 0-0.1 0-0.3-0.1-0.4-0.6-0.7-1-2-1-3.2 0-3.1 2.3-6 6.3-6 3.4 0 5.8 2.3 5.8 5.6 0 3.8-1.9 6.4-4.4 6.4-1.4 0-2.4-1.1-2-2.5 0.4-1.6 1.2-3.4 1.2-4.6 0-1.1-0.6-1.9-1.8-1.9-1.4 0-2.5 1.4-2.5 3.4 0 1.2 0.4 2.1 0.4 2.1s-1.4 5.8-1.6 6.9c-0.3 1.2-0.2 2.9 0 4-5.2-2-8.8-7-8.8-12.9 0-7.7 6.2-13.9 13.9-13.9s13.9 6.2 13.9 13.9c0 7.7-6.2 13.9-13.9 13.9z"/></svg>
+                  <li><a href="http://www.linkedin.com/company/10701?trk=tyah" target="_blank">
```

Comment:
Social icons didn't match production, and one of the AC involved them so I made cascade assets match.